proto/buf/alpha/registry/v1alpha1/role.proto

```diff
@@ -38,6 +38,7 @@ enum RepositoryRole {
   REPOSITORY_ROLE_ADMIN = 2;
   REPOSITORY_ROLE_WRITE = 3;
   REPOSITORY_ROLE_READ = 4;
+  REPOSITORY_ROLE_WRITE_DRAFTS = 5;
```

Comment:
I was thinking about if this need a comment as it's not as self described as others (and if we do that we should probably do it to the entire file), but that might be more like an implementation detail so I think it's good for now